department/models.py

```diff
@@ -10,3 +10,11 @@ class Department(models.Model):
 
     def __str__(self):
         return self.department
+
+
+class Notice(models.Model):
+    topic = models.CharField(max_length=100, default="Emergency Notice")

```

Comment:
does nullable works?

Reply:
BROTHER Have you done this job in laravel?

Reply:
No we have not done this in laravel.